src/server/helpers.py

```diff
@@ -195,6 +192,29 @@ def accentless_sort(value):
     return sorted(value, key=lambda i: strip_accents(i[1]).lower())
 
 
+def get_file_date_info(file, type):
+    timestamps_config = get_timestamps_config()
+    # Default Published and Last Updated to today
+    today = datetime.datetime.utcnow().isoformat()
+    if type == 'date_published' or type == 'date_modified':
+        return timestamps_config.get(file, {}).get(type, today)
+    else:
+        return timestamps_config.get(file, {}).get(type)
+
+
+def get_versioned_filename(path):
+    version = get_file_date_info(path, 'hash')
+    if version:
+        return '%s?v=%s' % (path, version)
+    else:
```

Comment:
Should this ever happen? If not, should it fail more loudly as an exception? For example, if it means that the frontend is trying to render a non-existent static resource that will inevitably 404, an exception will help us catch it during testing.

Reply:
When you add a new CSS file but haven’t run the pre-release script then if won’t exist in config yet. So that’s one instance. In prod hopefully shouldn’t happen.

Reply:
OK added an exception logging and confirmed it still returns fine so won't interrupt development.